src/sv-pipeline/04_variant_resolution/scripts/split_variants.py

```diff
@@ -0,0 +1,89 @@
+#!/bin/python
+import argparse
+import logging
+
+
+# Function to process the bed file by checking for conditions
+def process_bed_file(input_bed, n_per_split, bca=True):
+    svtype_field = 4
+    end_pos = 2
+    start_pos = 1
```

Comment:
Small style thing - since these are constants they should be in all caps. I would probably call end_pos `END_FIELD` and start_pos `START_FIELD` as well so that it doesn't seem like the actual coordinates are 1 and 2